src/ImageProcessor.Web.Plugins.PostProcessor/PostProcessor.cs

```diff
@@ -34,168 +37,135 @@ internal static class PostProcessor
         /// <param name="stream">The source image stream.</param>
         /// <param name="extension">The image extension.</param>
         /// <returns>
-        /// The <see cref="MemoryStream"/>.
+        /// The <see cref="MemoryStream" />.
         /// </returns>
-        public static MemoryStream PostProcessImage(HttpContext context, MemoryStream stream, string extension)
+        public static async Task<MemoryStream> PostProcessImageAsync(HttpContext context, MemoryStream stream, string extension)
```

Comment:
There are use-cases for making the PostProcessor public, e.g. for optimizing uploaded images. In that case, we should probably ditch the `HttpContext` (or make it optional), because it's only used for logging...